tools/azure-pipelines/jobs-template.yml

```diff
@@ -162,13 +162,18 @@ jobs:
       displayName: Setup Docker
     - script: ./tools/azure-pipelines/free_disk_space.sh
       displayName: Free up disk space
+    - script: sudo apt-get install -y bc
```

Comment:
what's this for?

Reply:
One of the tests which are running when S3 is preset require `bc`

Reply:
It's called in `common_s3_operations.sh`.